src/lib/utils/strings.ts

```diff
@@ -20,24 +20,18 @@ export const getStringFromLeft = (str: string, length: number): string => {
     return str.substring(0, length);
 };
 
-export function randomIntFromInterval(min: number, max: number): number {
+/**
+ * @internal
+ */
+export function randomIntFromRange(min: number, max: number): number {
```

Comment:
Create a new file called `utils/random.ts` and move this method to it. After all, ints have nothing to do with strings.